monorepo/website/src/pages/news/2024-11-01-autumn-update.mdx

```diff
@@ -0,0 +1,51 @@
+---
+layout: ../../layouts/MdLayout.astro
+title: "Pathoplexus Autumn Update"
+---
+
+<div>
+    <a href="/news">News</a> > <b>Pathoplexus Autumn Update</b>
+</div>
+
+# Pathoplexus Autumn Update
+
+*By the Pathoplexus Team - 1 November 2024*
+
+Despite only being 2 months old, we at Pathoplexus have been excited to already contribute to discussions on data sharing, and start becoming part of the pathogen sharing community (see 'past events' below). In addition, we've continued to add new features and improvements to Pathoplexus - still largely powered by volunteer effort and donated time!
+
+As the seasons turn, we wanted to share some of our recent news:
+
+### In the media:
+- [Lancet Microbe's Elena Dalla Vecchia covered Pathoplexus](https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(24)00263-5/fulltext), highlighting how Pathoplexus is aiming to improve data sharing through openness, credit, and transparency (13 Sept 2024)
+
+### Visitors and data:
+- Since launch, Pathoplexus has automatically ingested 53 newly released sequences from INSDC-member databases
+- We are excited to have hit 100 registered users! (Necessary to upload data but not to view data)
+
+### Executive Board update:
+- See recent minutes of [Pathopexus Members meetings, Executive Board meetings, and Executive Board resolutions](https://pathoplexus.org/about/governance/minutes)
+
+### Scientific Advisory Board:
+- We’re incredibly pleased to welcome Meera Chand from the UKHSA to the [Scientific Advisory Board](https://pathoplexus.org/about/sab)!
+
+### Pathoplexus Association:
+- The Pathoplexus association has been recognized as tax-exempt in Switzerland. This means any donations made by Swiss residents are now tax-deductible.
+
+### Software updates:
+- You can now use checkboxes to select exactly the sequences you'd like to download from the search/browse pages
+  ![GIF showing how to select sequences for download](/gifs/news/2024-11-01-autumn-update/checkbox.gif)
+- It's now possible to download both the sequence and metadata for individual sequences easily from their 'detail view' page
+- Group names and information is now easily editable
+- Multi-sequence downloads now have detailed file names, featuring the organism, date, and data type (amino-acid, aligned nucleotide, etc)
+- We've improved how we store author names so that we can more flexibly display them, and pass them in preferred formats on when depositing to INSDC
```

Comment:
This line sounded a bit odd to me when reading through, should it not be
```suggestion
- We've improved how we store author names so that we can display them more flexibly, and pass them on in preferred formats when depositing to INSDC
```

Reply:
I think this isn't really a feature, it's a breaking change for INSDC compatibility reasons.

Ok there is a feature and it's that we normalize name now to be sentence case from INSDC, but this is sort of unrelated to the reformatting breaking change